.ci-dockerfiles/centos7-llvm-3.9.1/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:7
+
+ENV LLVM_VERSION 3.9.1
+
+RUN yum install -y \
+    yum-plugin-copr \
+    which \
+    git \
+    gcc-c++ \
+    make \
+    openssl-devel \
+    pcre2-devel \
+    zlib-devel \
+    ncurses-devel \
+    libatomic \
+ && yum copr enable -y alonid/llvm-${LLVM_VERSION} \
+ && yum install -y \
+    llvm-${LLVM_VERSION} \
+    llvm-${LLVM_VERSION}-devel \
+    llvm-${LLVM_VERSION}-static
```

Comment:
Would you mind adding these few lines to ensure the ci tests are not run as root? This is necessary for some stdlib tests of the `files` package i am working on in #2656 .

```
RUN useradd -ms /bin/bash -d /home/pony -g root pony
USER pony
WORKDIR /home/pony
```

Reply:
Done.